etc/chromeos/lib/installs.sh

```diff
@@ -268,24 +279,16 @@ install_hashicorp_tools() {
     echo_banner "Installing hashicorp tools: consul nomad packer"
 
     # Set specific versions since we're enabling the hashicorp test repo
-    CONSUL_VERSION="1.12.0"
+    CONSUL_VERSION="1.12.0-1"
     NOMAD_VERSION="1.3.0-1~beta.1"
     PACKER_VERSION="1.8.0"
     VAULT_VERSION="1.10.0"
```

Comment:
see https://github.com/grapl-security/grapl/pull/1695

Reply:
@christophermaier I just merged #1695 in (I had forgotten I hadn't done that yet). That should fix the versions once you've merged this in and dealt with the conflict.

Reply:
Ah, cool 👍